Src/Couchbase.Linq/Couchbase.Linq.csproj

```diff
@@ -81,6 +81,7 @@
     <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.Net.Http.WebRequest" />
```

Comment:
We are, this is a bit of an oddity in HttpClient.  It is a set of extensions for HttpClient that adds some of the missing features that WebRequest had.  Specifically in this case, the ability to add a custom certificate validator.

If we add a .Net Core version of the LINQ client eventually, we may have to reevaluate because this assembly doesn't currently have a .Net Core build.  But I think Relinq is going to be our biggest blocker there.